types/ultron/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for ultron 1.1
+// Project: https://github.com/unshiftio/ultron
+// Definitions by: Renée Kooi <https://github.com/goto-bus-stop>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.7
+
+type Listener = (...args: any[]) => void;
+
+/**
+ * A Node.js EventEmitter instance or an `eventemitter3` instance.
+ */
+interface EventEmitter {
+    on(event: string | symbol, fn: Listener): void;
+    once(event: string | symbol, fn: Listener): void;
+    removeListener(event: string | symbol, fn: Listener): void;
+    listeners(event: string | symbol): any[];
```

Comment:
Ideally yes, but the typings for `EventEmitter` in the `@types/node` package currently return `Function[]` instead. I just used `any[]` to avoid the lint warnings about `Function[]`, the actual type doesn't really matter since `ultron` just needs to `===` it.

Reply:
Works for me.  Thanks!